config/environments/test.rb

```diff
@@ -74,4 +74,10 @@
     end
     ASDeprecationTracker.resume!
   end
+
+  config.after_initialize do
+    Bullet.enable = true
+    Bullet.bullet_logger = true
+    Bullet.raise = true # raise an error if n+1 query occurs
```

Comment:
It's a bit harsh, since bullet has a couple of false positives. So we will need to white-list those.

Reply:
That's why I opened it as a draft, but of course those logs have been purged by now. I've rebased it to see how bad it is now.